packages/tizen_rpc_port/lib/src/port.dart

```diff
@@ -0,0 +1,73 @@
+import 'dart:async';
+import 'dart:typed_data';
+
+import 'parcel.dart';
+import 'rpc_port_method_channel.dart';
+
+/// The port type.
+enum PortType {
+  /// The main port type.
+  main,
+
+  /// The callback port type.
+  callback,
+}
+
+/// The port connection between proxy & stub.
+class Port {
+  /// Create port from proxy.
+  Port.fromProxy(this.portName, this.instance, this.portType) : appid = '';
+
+  /// Create port from stub.
+  Port.fromStub(this.appid, this.portName, this.portType) : instance = '';
+
+  /// The type of port[main|callback].
+  final PortType portType;
+
+  /// The appid of stub app. This member is used only proxy.
+  final String appid;
+
+  /// The port name of port connection.
+  final String portName;
+
+  /// The instance name of proxy. This member is used only stub.
+  final String instance;
+
+  /// Sends a parcel to connected app.
+  Future<void> send(Parcel parcel) async {
+    final MethodChannelRpcPort manager = MethodChannelRpcPort.instance;
```

Comment:
You may create a private field for this and remove all local variable `manager` in several methods. Also `_methodChannel` seems like a better name.